ui/numbered_menu.py

```diff
@@ -0,0 +1,141 @@
+from threading import Lock
+from time import sleep, time
+
+from ui import Menu
+from ui.base_list_ui import BaseListUIElement
+from ui.numpad_input import check_value_lock
```

Comment:
`from ui.utils import`?

Reply:
I don't understand your feedback. Can you elaborate ?

Reply:
The `check_value_lock` function is no longer located in `ui.numpad_input`, but in `ui.utils`, so while the import works, it's confusing =)